lib/components/Toggle/Toggle.tsx

```diff
@@ -48,6 +48,8 @@ export const Toggle = ({
         checked={on}
         onChange={handleChange(onChange)}
         position="absolute"
+        cursor="pointer"
```

Comment:
Ah, just saw this. Does this still apply when it's disabled?

Reply:
You can't disable a toggle

Reply:
Didn't catch your earlier comment. Please ignore me.